woodwork/tests/logical_types/test_logical_types.py

```diff
@@ -243,3 +246,29 @@ def test_url_validate(sample_df):
     actual = logical_type.validate(series, return_invalid_values=True)
     expected = pd.Series({4: "bad_url"}, name="url")
     assert to_pandas(actual).equals(expected)
+
+
+@pytest.mark.parametrize(
+    argnames="logical_type",
+    ids=["age", "age_fractional", "age_nullable"],
+    argvalues=[Age(), AgeFractional(), AgeNullable()],
+)
+def test_age_validate(sample_df, logical_type):
+    invalid_row = pd.Series({4: -3}, name="age", dtype="Int64")
+    series = sample_df["age"]
+
+    if _is_koalas_series(series):
+        invalid_row = ks.from_pandas(invalid_row)
+    assert logical_type.validate(series, return_invalid_values=False) is None
+
+    series = series.append(invalid_row).astype("Int64")
+    if isinstance(logical_type, AgeFractional):
```

Comment:
With the change in this spot we aren't testing the first assertion with a `float64` dtype.

Reply:
updated